.github/workflows/cdk-diff.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           result="${result//'%'/'%25'}"
           result="${result//$'\n'/'%0A'}"
           result="${result//$'\r'/'%0D'}"
-          echo "::set-output name=result::${result}"
+          echo "result=${result}" >> $GITHUB_OUTPUT
```

Comment:
https://github.com/dev-hato/youtube_streaming_watcher/actions/runs/3243564359/jobs/5318459533

```
.github/workflows/cdk-diff.yml:61:9: shellcheck reported issue in this script: SC2086:info:8:28: Double quote to prevent globbing and word splitting [shellcheck]
   |
61 |         run: |
   |         ^~~~
```

他もおそらく同様。

Reply:
```suggestion
          echo "result=${result}" >> "${GITHUB_OUTPUT}"
```